<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSimCtrl</class>
 <widget class="QWidget" name="ImageSimCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageSimCtrl</string>
  </property>
  <widget class="QLabel" name="label_DataPath">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>411</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DataPath</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_startSim">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>StartImgSim</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>213</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_pixelDiff">
    <property name="text">
     <string>pixelDiff</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pixelDiff"/>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>96</y>
     <width>213</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_bloclCnt">
    <property name="text">
     <string>blockCnt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_blockCnt"/>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>123</y>
     <width>213</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_perThread">
    <property name="text">
     <string>perThread</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_perThread"/>
  </widget>
  <widget class="QPushButton" name="pushButton_Differ">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>startDiffer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DeteSim">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>StartObjDetetionAlgorithm</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>225</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_trSize">
    <property name="text">
     <string>tolerantSize</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_trSize"/>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>127</y>
     <width>225</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_threshold">
    <property name="text">
     <string>threshold</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_threshold"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
